### PR TITLE
Added IPv6 excuses

### PR DESCRIPTION
**Pull request by cazagen**

### CODE REVIEW

File: ircbot/plugins/excuses.py [hunk 1]
```diff
@@ -266,8 +266,66 @@
...
+    "We'll deploy IPv6 right after we deploy DNSSEC",
+    "It'll break our GeoIP",
+    "Our business intelligence team can't even parse IPv4 logfiles",
+    "Our DDOS migation platform can't monitor IPv6",
+    "We forgot to include IPv6 in our last RFP",
+    "Did you mean IPTV?",
+    "We have no roadmap for native IPv6 as we rolled out 6RD",
+    "Too many people have broken IPv6 stacks",
+    "We have IPv6, but we just want to keep things simple",
+    "Our Dynamic DNS doesn't support it"]
```
Comment (neanias) [inline ircbot/plugins/excuses.py]: I'm thinking "We're holding out for IPv10. It's the next big thing."

Comment (cazagen) [reply ircbot/plugins/excuses.py]: That can be done

Comment (neanias) [reply ircbot/plugins/excuses.py]: Called it: https://twitter.com/ipv6excuses/status/905828983849324544

**Review by neanias — APPROVED** [review]
`{bot.trigger}shipit`

**Comment by hreeder** [comment]
I'd love to see this just use the existing `excuse` command with maybe an argument, ie `excuse ipv6`.

We'd be able to store the keywords in a dict, which would maybe look like this:
```python
{
    "default": excuses,
    "ipv6": ipv6excuses
}
```

**Comment by cazagen** [comment]
I'll get on that at some point

**Pull request merged**